Source/Atomic/UI/UIMenuWindow.cpp

```diff
@@ -83,4 +94,16 @@ void UIMenuWindow::Close()
 
 }
 
+void UIMenuWindow::FixMenu( tb::TBMenuWindow *mymenu ) // get rid of scrollbars for menu popups
+{
+    if ( mymenu )
+    {
+        TBScrollContainer *mys = mymenu->GetList()->GetScrollContainer();
+        mys->SetScrollMode(SCROLL_MODE_OFF);
+        mys->SetAdaptToContentSize(true);
+        mys->SetAdaptContentSize(false);
```

Comment:
Ah yes, the adapt content dance, we need a better idiom... this has cost a few minutes here as well :)